docs/Docker/dockerLocal_prodVault.md

```diff
@@ -0,0 +1,126 @@
+# Run MPS and RPS using docker and production vault
+## Introduction
+This document describes how to run MPS and RPS using vault in production server mode. The current local docker-compose file runs vault in development mode which makes experimenting with the services easier since static tokens can be used for access and unsealing vault is not required. The downside to this approach is all vault data is stored only in memory which is lost once the vault container is stopped. Running vault in production mode requires more steps but allows vault data to persist after container restarts.
+
+
+1\. Update the vault section in the docker-compose file (scripts\docker-compose\docker-compose.yaml) with the section below: 
+
+```
+ vault:
+    image: vault
+    container_name: prodvault
+    environment:
+      VAULT_ADDR: http://127.0.0.1:8200
+    ports:
+      - "8200:8200"
+    volumes:
+      - private-volume:/vault/file:rw
+      - ./vault:/vault/config:rw
+    cap_add:
+      - IPC_LOCK
+    entrypoint: vault server -config=/vault/config/vault.json
+
+```
+
+2\. Create a folder named 'vault' located in scripts\docker-compose\ and add a file named vault.json with the contents below:
```

Comment:
Looking at entrypoint above, it seems like this instruction should say "Create a folder path named '/vault/config/' and add a file named vault.json..."

Are these pointing referencing the same place?

Reply:
the ./vault in the volumes section is the folder that is mounted from the host system... that maps to /vault/config in the container. Sorry about the confusion.

Reply:
Ahh, ok.  Figured it was something like this.  I approve then 👍